### PR TITLE
Enhancement/list footer

### LINKED ISSUE #347: [Enhancement] list footer
**Is your enhancement request related to a problem? Please describe.**
Having content, below a list on native, is right now not possible due to the nature/scrollability of the native list. 

**Describe the solution you'd like**
To place content below a list, a footer implementation will be implemented by using a directive to load the elements below the list. This will be done utilising the RadListViews tkListViewFooter directive. That way content should be scrollable, and be displayed properly below a list. 

**Describe alternatives you've considered**
An alternative to this, would be to design pages without content below the list, as is with most other applications.

**Additional context**
https://docs.nativescript.org/angular/ui/professional-ui-components/ng-ListView/header-footer

### PR DESCRIPTION
**Pull request by PetarRing**
closes #347

### CODE REVIEW

File: src/app/examples/nativescript-only/list-header-and-footer/list-header-and-footer-example.component.tns.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<kirby-list [items]="items"
+        shape="rounded">
```
Comment (jakobe) [inline src/app/examples/nativescript-only/list-header-and-footer/list-header-and-footer-example.component.tns.html]: Shape is now rounded by default:
```suggestion
        >
```

File: src/app/examples/nativescript-only/list-header-and-footer/list-header-and-footer-example.component.tns.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        <StackLayout *kirbyListHeader>
+            <Label text="This is the header" textWrap="true" class="header"></Label>
+        </StackLayout>
+
+  <kirby-list-flex-item *kirbyListFlexItem="let item">
```
Comment (jakobe) [inline src/app/examples/nativescript-only/list-header-and-footer/list-header-and-footer-example.component.tns.html]: Simplify the example by removing `<kirby-list-flex-item>` and `<kirby-list-cell>` wrappers and just applying `*kirbyListItem` directive to a `StackLayout` itself (as you already do for header/footer):
```suggestion
  <StackLayout *kirbyListItem>
          <Label [text]="item.title"></label>
        </StackLayout>
```

File: src/app/examples/nativescript-only/list-header-and-footer/list-header-and-footer-example.component.tns.html [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    <kirby-list-cell>
+        <GridLayout columns="*" rows="auto">
+          <kirby-list-cell-line col="0" row="0" [primary]="true" [text]="item.title"></kirby-list-cell-line>
+        </GridLayout>
+      </kirby-list-cell>
+</kirby-list-flex-item>
+
+<StackLayout *kirbyListFooter>
+    <Label text="This is the footer" textWrap="true" class="header"></Label>
```
Comment (jakobe) [inline src/app/examples/nativescript-only/list-header-and-footer/list-header-and-footer-example.component.tns.html]:
```suggestion
    <Label text="This is the footer" textWrap="true" class="footer"></Label>
```

File: src/app/examples/nativescript-only/list-header-and-footer/list-header-and-footer-example.component.tns.scss [hunk 4]
```diff
@@ -0,0 +1,4 @@
+.header {
```
Comment (jakobe) [inline src/app/examples/nativescript-only/list-header-and-footer/list-header-and-footer-example.component.tns.scss]:
```suggestion
.header, .footer {
```

File: src/app/examples/nativescript-only/list-header-and-footer/list-header-and-footer-example.component.tns.html (hunk 2 above)
Comment (PetarRing) [reply src/app/examples/nativescript-only/list-header-and-footer/list-header-and-footer-example.component.tns.html]: Isn't `*kirbyListItem` going to be deprecated? Or are we going to differentiate on `*kirbyListItem` and `*kirbyListFlexItem` as such?

Comment (jakobe) [reply src/app/examples/nativescript-only/list-header-and-footer/list-header-and-footer-example.component.tns.html]: It's a left over from the performance issue just before the release - but no, `*kirbyListItem` is not going to be deprecated :)

**Pull request merged**